service/coupon/src/main/resources/application.yml

```
@@ -0,0 +1,26 @@
+
+  datasource:
+    url: ${DB_URL}
+    username: ${DB_USERNAME}
+    password: ${DB_PASSWORD}
+    driver-class-name: com.mysql.cj.jdbc.Driver
+  jpa:
+    hibernate:
+      ddl-auto: create
+    properties:
+      hibernate:
+          show_sql: true # sql 로깅
```

yamllint

[warning] wrong indentation: expected 8 but found 10

(indentation)